addons/base_automation/static/src/js/crash_manager.js

```diff
@@ -0,0 +1,38 @@
+odoo.define('base_automation.CrashManager', function (require) {
+  "use strict";
+  var CrashManager = require('web.CrashManager');
+  var ErrorDialog = CrashManager.ErrorDialog;
+  ErrorDialog.include({
```

Comment:
In general, we try to avoid `include`. Here is a suggestion using `extend` instead:
   - have an `ErrorDialogRegistry` with keys being error types (or any other optional error attribute)
   - in `show_error` of `CrashManager`, check if there is a specific `ErrorDialog` in the registry to the given error, and if so, use this one ; otherwise, use the generic `ErrorDialog`
   - in `base_automation`, create your specific `BaseAutomationErrorDialog` extending `ErrorDialog` with your custom behavior (and template).

With this design, you could probably give the whole error to the `ErrorDialog`. That way, you'll get rid of the `options` thing that was bothering you.

Reply:
I will give it a try. I wanted as well to avoid the include so the new template is not used when it's not needed but I didn't know how to proceed / what was your way to do it. Now I have a hint :). Thanks !